From ecaa26f65d0f919cab76f9aebf56e4ae075f1160 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Tue, 18 Jul 2017 00:52:34 +0800
Subject: [PATCH 2231/5242] MLK-16013-6 Documentation: usb: dwc3: add suspend
 clk setting

commit  f08bb396f0e6d1f69c2c229c996b03581c6b6a56 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add dt documentation for specify the suspend clk and its caculation.

Reviewed-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 Documentation/devicetree/bindings/usb/dwc3.txt |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/usb/dwc3.txt b/Documentation/devicetree/bindings/usb/dwc3.txt
index 7f13ebe..d55597a 100644
--- a/Documentation/devicetree/bindings/usb/dwc3.txt
+++ b/Documentation/devicetree/bindings/usb/dwc3.txt
@@ -94,6 +94,9 @@ Optional properties:
 			this and tx-thr-num-pkt-prd to a valid, non-zero value
 			1-16 (DWC_usb31 programming guide section 1.2.3) to
 			enable periodic ESS TX threshold.
+- snps,power-down-scale: Value for number of scale(16kHz) of suspend clock if
+        the default value is not correct, valid range is 2~8000, means suspend
+        clock is from 32KHz to 125MHz.
 
  - <DEPRECATED> tx-fifo-resize: determines if the FIFO *has* to be reallocated.
 
-- 
1.7.9.5

